src/components/csvLocalizer/jsonLocalizer.js

```diff
@@ -48,14 +48,32 @@ const JsonLocalizer = () => {
 
   const onCancel = React.useCallback(() => {
     setActiveCellId('');
-    setNewtranslation('');
+    setAlert('');
   }, []);
 
+  const activeCellContent =
+    activeCellId &&
+    data[activeCellId.split('-')[0]][+activeCellId.split('-')[1]];
```

Comment:
This line is hardly readable.
I'm sure you know what to expect from the activeCellId, but first what I thought of was 'what if this expression will return undefined? we'll get the activeCellContent equal to the activeCellId'.
let activeCellContent;
const [x, y] = activeCellId?.split('-');
if (x && y) activeCellContent = data[x][y];

Reply:
edited